discordrb.gemspec

```diff
@@ -44,4 +44,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop-rake', '~> 0.6.0'
   spec.add_development_dependency 'simplecov', '~> 0.21.0'
   spec.add_development_dependency 'yard', '~> 0.9.9'
+  spec.add_development_dependency 'pry-byebug'
```

Comment:
I don't think this dependency is really needed, but, if there is, feel-free to start a discussion in the [drb channel.](https://discord.gg/discord-api)